### ISSUE TITLE
Window size differs from process value differences

### ISSUE THREAD

**Issue by Jamire** [caoyue/WindowResizer#72]
Hi,
I want to use WindowsResizer to resize a window to a specific size because I need to take screenshots with that size. So I edited the values for Top, Left, Right, Bottom so that the differences (Bottom - Top) and Right - Left) match the desired size.
However, there is always a difference of 14 for Bottom-Top and 7 for Right-Left. Maybe these are fixed values due to the additional width of the window frame or something like that. Or is it a bug? Still, it would be nice to be able to just enter the real values without any calculations.
Possibly directly as width and height value and not as differences?

**Comment by caoyue** [comment]
Yes, the window size is affected by Window Border and other factors, and the API provided by Windows is also based on position rather than width and height.
I'll look into this to see if it can be implemented.
